### PR TITLE
add recommendation_update

### PR DESCRIPTION
**Pull request by adelaidemz**

### CODE REVIEW

File: src/backend/server/api/handlers/recommendation_update.py [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+        rec = Recommendation.objects.filter(id=recoID)
+    except Exception:
+        return HttpResponseServerError("cannot find recommendation")
+
+    try:
+        # with connection.cursor() as cursor:
+        #   cursor.execute(CHECK_QRY.format(recoID) )
+        rec.seen_count = rec.seen_count + 1
+        for k in sr.body:
+            if k == "accepted":
```
Comment (simaoalicechen) [inline src/backend/server/api/handlers/recommendation_update.py:30]: This makes sense to me.

File: src/backend/server/api/handlers/recommendation_update.py [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+    try:
+        recoID = sr.body.get("recoID", None)
+        rec = Recommendation.objects.filter(id=recoID)
+    except Exception:
+        return HttpResponseServerError("cannot find recommendation")
+
+    try:
+        # with connection.cursor() as cursor:
+        #   cursor.execute(CHECK_QRY.format(recoID) )
+        rec.seen_count = rec.seen_count + 1
```
Comment (simaoalicechen) [inline src/backend/server/api/handlers/recommendation_update.py:28]: This looks good

**Review by simaoalicechen — COMMENTED** [review]
The changes make sense me. What makes you think they will work in develop? (I know it works remotely). Does it have anything to do with the fact that the previous pr fixed the lint tests and having the new school list api ?

**Comment by adelaidemz** [comment]
> The changes make sense me. What makes you think they will work in develop? (I know it works remotely). Does it have anything to do with the fact that the previous pr fixed the lint tests and having the new school list api ?

I actually deployed an earlier iteration of the API endpoint, and was able to hit against it. Then made some modifications so that it operated correctly. It's not related to the previous PR, that was only fixing the issue with the JS tests and JS linting.

**Comment by simaoalicechen** [comment]
I don't think this has major problems, but let's talk tomorrow before merge.

**Comment by simaoalicechen** [comment]
This is working in Production (on AWS side).  It appears to be not passing the tests. Given Daniel's comment about the tests passing if pr is from team repo, it most likely would be a problem with the GitHub actions, instead of the codes. We will revert this, if this pr does not work out as expected.

**Pull request merged**